index.js

```diff
@@ -36,6 +36,7 @@ client.on('message', async msg => {
             firstTime = true;
         }
         if (!firstTime && msg.createdTimestamp < user.timestamp) {
+            await msg.author.send("Your message was deleted. You need to wait " + ((user.timestamp - msg.createdTimestamp)/1000).toString() + " more seconds.\n > " + msg.content);
```

Comment:
This lets the user know (in seconds) how much time they need to wait before posting again. It also quotes their deleted message so they can copy and paste it for the next try.